lighthouse-core/gather/gatherers/offline.js

```diff
@@ -25,7 +26,8 @@ class Offline extends Gatherer {
 
   afterPass(options, tracingData) {
     const navigationRecord = tracingData.networkRecords.filter(record => {
-      return record._url === options.url && record._fetchedViaServiceWorker;
+      return cleanHash(new URL(record._url)) === cleanHash(new URL(options.url)) &&
```

Comment:
I'm not sure I understand the full context, but in general, yeah, I think ignoring the `hash` would make sense from the point of view of both the Cache Storage API and [navigation requests](https://fetch.spec.whatwg.org/#navigation-request).